src/test/ui/reject-specialized-drops-8142.rs

```diff
@@ -1,6 +1,9 @@
 // Issue 8142: Test that Drop impls cannot be specialized beyond the
 // predicates attached to the type definition itself.
 
+#![feature(custom_inner_attributes)]
+#![rustfmt::skip]
```

Comment:
Why would this be needed?

Reply:
I just didn't want to format the test in this PR and my IDE kept autoformatting it. Apparently `./x.py fmt` does not format tests

Reply:
I removed this in the other PR, once that merges this PR won't change this file anymore